app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/registration/SubscriptionRecoveryController.scala

```diff
@@ -97,6 +98,11 @@ class SubscriptionRecoveryController @Inject() (
           safeId,
           Eori(eori),
           subscriptionDisplayResponse,
+          getDateOfBirthOrDateOfEstablishment(
+            subscriptionDisplayResponse,
+            registrationDetails.dateOfEstablishmentOption,
+            registrationDetails.dateOfBirthOption
+          ),
```

Comment:
As I remember here they use only subscription details, not sure why, but in general it shouldn't be a problem, because subscription details will always has a priority

Reply:
https://github.com/hmrc/customs-rosm-frontend/commit/88cb8742a5dadd8f207e398b73a18a513115568e#diff-9f7e64f4cff585849c26665d40c3edb933c44e093d2d3cf2f62f1a50dc223d88R110 The link for above

Reply:
You are correct - good spot.  But if the date can be missing on subscription journey, why not on register journey?

Reply:
Agree with you.

So I think we can keep this, because or prioritisation SUB09 date